subjects/matrix_multiplication/README.md

```diff
@@ -0,0 +1,53 @@
+## matrix_multiplication
+
+### Instructions
+
+- Define a `struct` named `Matrix` as a tuple of 2 tuples. The nested tuple will contain 2 `i32`s.
+
+- Create a **function** named `multiply` that receives a Matrix and an i32 and returns the Matrix with each number multiplied by the second argument. 
+```rust
+pub fn multiply(m: Matrix, val: i32) -> Matrix {
```

Comment:
`val` could be `multiplier` or something like that, as soon as it is more informative.
As a general rule it is better to avoid names like `val` which are highly uninformative and may suggest bad practices to students.